db/migrate/20230222101705_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true, index:true
```

Comment:
- [ ] According to the ERD diagram it is required the user to be named author so I would recommend you change this t.reference to:
`    t.references :author, foreign_key: { to_table: 'users' }`